Trigger/src/ReadTriggerInfo_module.cc

```diff
@@ -83,14 +83,14 @@ namespace mu2e {
       kNTrigInfo     = 40,
       kNTrackTrig    = 20,
       kNTrackTrigVar = 50,
-      kNHelixTrig    = 10,
+      kNHelixTrig    = 40,
```

Comment:
When you change these numbers, do you need to make the corresponding changes in other parts of the code or in fcl or config files?  If so, what would it take to get a single point of maintenance?

Reply:
These numbers cannot be part of the list of data-members of the class, because they define the size of the arrays of histogram I set in my `struct` instances. I usually change it when the Menu I test gets longer, which happens not that frequently.